openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1987,13 +1989,7 @@ The following additional Credential Issuer metadata parameters are defined for t
 
 * `credential_definition`: REQUIRED. Object containing the detailed description of the Credential type. It consists of at least the following two parameters:
```

Comment:
Isn't the sentence "It consists of at least the following two parameters" misleading because `credentialSubject` is marked as optional, so a valid `credential_definition` may also only contain one parameter?

I would suggest something like "It consists of at least the type parameter and an optional credentialSubject".

Reply:
This text is no longer present in the new version, so I'll close this.